R/make_split_fun.R

```diff
@@ -296,14 +307,14 @@ make_split_fun <- function(pre = list(), core_split = NULL, post = list()) {
 
     if (is.null(core_split)) {
       ret <- do_base_split(spl = spl, df = df, vals = vals, labels = labels)
-    } else if (!in_col_split(.spl_context)) {
+    } else if (TRUE||!in_col_split(.spl_context)) {
```

Comment:
```suggestion
    } else if (TRUE || !in_col_split(.spl_context)) {
```
If there is no stop after the else isnt it better to have this as final else? I think it could be better to add a check for sub_expr instead when `in_col_split(.spl_context)` is true, right?

Reply:
It should actually just be an else, I'll fix this in a new commit